.github/workflows/check.yml

```
@@ -35,8 +35,8 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           cache: pip
-          cache-dependency-path: pyproject.toml
-      - run: pip install -e .[dev]
+          cache-dependency-path: requirements-dev.txt
+      - run: pip install -r requirements-dev.txt
```

```suggestion
          cache-dependency-path: pyproject.toml
      - run: pip install -e .[dev]
```
maybe I should go to sleep instead of trying to fix merge conflicts